lib/workos/configuration.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# typed: false
+
+module WorkOS
+  # Configuration class sets config initializer
+  class Configuration
```

Comment:
Can we add some test coverage for this new class? Just something basic, to ensure that `timeout`, `key`, and `key!` work as expected, since they are a part of the core behavior of the gem.

Reply:
Definitely :) I added some tests - let me know if there is missing coverage 👍